### DISCUSSION TITLE
[!DANGER!]failed to forbid an old account from accessing to a github respository through github desktop

### DISCUSSION THREAD

**Discussion by WenTianShi** [community/community#24092] [Apps, API and Webhooks]
<p>I tried to forbid an old account from accessing to a github respository , but totally failed.<br>
I Have tried the ways below ,but the GitHub Desktop which login three weeks ago can still access the respository totally with pull, push,commit, etc.<br>
(1)change the password of the github account through Chrome Browser. The  GitHub Desktop which login three weeks ago can still access the same respository without update the password.<br>
(2)change the ownership of the respository from one to another new one. But the GitHub Desktop which login three weeks ago can still access the respository without changing anything (git url is old, password is old, but can still access the respository totally with pull, push,commit, etc.)</p>

**Comment by RodnyPHP** [comment]
<aside class="quote no-group" data-username="WenTianShi" data-post="1" data-topic="151079">
<div class="title">
<div class="quote-controls"></div>
 WenTianShi:</div>
<blockquote>
<p>I Have tried the ways below ,but the GitHub Desktop which login three weeks ago can still access the respository totally with pull, push,commit, etc.</p>
</blockquote>
</aside>
<p>We can response to the Admin pannel, we have made Staff power</p>

**Comment by steveward** [comment, marked as answer]
<p><a class="mention" href="/wentianshi">@WenTianShi</a> welcome, and thanks for the question.</p>
<p>GitHub Desktop uses an OAuth token which allows it to access your GitHub account. This OAuth token is still valid even if the password associated with your GitHub account changes.</p>
<p>You can revoke OAuth applications by going to <a href="https://github.com/settings/applications">https://github.com/settings/applications</a> – see <a href="https://github.com/settings/applications">this article</a> for more information. GitHub has a <a href="https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/updating-your-github-access-credentials">full guide</a> for resetting all of the various access credentials in the case of your account being compromised.</p>
<p>I hope that helps! Let me know if you have any other questions about this.</p>
